views/inputs/datetime.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Datetime input template
+ *
+ * @since x.x.x
+ */
+
+declare( strict_types=1 );
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * @var \Automattic\WooCommerce\GoogleListingsAndAds\View\PHPView $this
+ */
+
+/**
+ * @var array $input
+ */
+$input = $this->input;
+
+$input['class']         = $input['class'] ?? '';
+$input['placeholder']   = $input['placeholder'] ?? '';
+$input['style']         = $input['style'] ?? '';
+$input['wrapper_class'] = $input['wrapper_class'] ?? '';
+$input['name']          = $input['name'] ?? $input['id'];
+$input['desc_tip']      = $input['desc_tip'] ?? false;
+$input['date']          = $input['date'] ?? '';
+$input['time']          = $input['time'] ?? '';
+
+// Custom attribute handling
+$custom_attributes = [];
+if ( ! empty( $input['custom_attributes'] ) && is_array( $input['custom_attributes'] ) ) {
+	foreach ( $input['custom_attributes'] as $attribute => $value ) {
+		$custom_attributes[] = esc_attr( $attribute ) . '="' . esc_attr( $value ) . '"';
+	}
+}
```

Comment:
Why do we add so many of these which we don't even use? Do we allow filtering input fields?

Reply:
I aimed to keep it similar to other WooCommerce inputs so that if we need to customize it further, we could easily do so. But I guess we could also add these customizations later if we ever need them so I've removed most of them in https://github.com/woocommerce/google-listings-and-ads/pull/951/commits/49b7409d19b8fc91688851fa66948ff1aceca0c5